app/views/admin/timeline/_step.html.slim

```diff
@@ -44,7 +44,7 @@
       - if index == latest_logs.length - 1
         div = link_to t("project_logs.more"), "#", data: {action: 'view-logs'}
 
-  td.step-menu-col class="dropup #{classes}" data-id=entry.id
+  td.step-menu-col class="dropup #{classes}" data-id=entry.id data-parent-id=entry.parent_id
```

Comment:
Note that I changed `parent.id` to `parent_id`. It's best to use the latter if it's possible since calling `parent.id` may trigger a database query to populate the `parent` object. If you need any other property of the parent, e.g. `summary`, you obviously have to call `parent.summary`, but with IDs you can often just do it directly.